llvm/lib/Target/SPIRV/SPIRVModuleAnalysis.cpp

```diff
@@ -1005,6 +1011,13 @@ void addOpAccessChainReqs(const MachineInstr &Instr,
   }
 }
 
+static bool imageTypeHasUnknownFormat(SPIRVType *TypeInst) {
+  if (TypeInst->getOpcode() != SPIRV::OpTypeImage)
```

Comment:
assert instead?

Reply:
I don't want too many asserts. I renamed to `isImageWithUnknownFormat`.